packages/flutter_test/lib/src/test_text_input.dart

```diff
@@ -6,25 +6,35 @@ import 'dart:async';
 import 'dart:convert';
 import 'dart:typed_data';
 
-import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/services.dart';
 
+import 'widget_tester.dart';
+
 const String _kTextInputClientChannel = 'flutter/textinputclient';
 
-class MockTextInput {
+/// A testing stub for the system's popup keyboard.
```

Comment:
nit: I think it's usually called an "onscreen" keyboard rather than a "popup" keyboard.